packages/ifpack2/test/unit_tests/Ifpack2_UnitTestBlockTriDiContainerUtil.hpp

```diff
@@ -246,7 +246,7 @@ struct BlockTriDiContainerTester {
           auto input = T_bare->createDefaultApplyParameters();
           input.zeroStartingSolution = zero_starting;
           input.maxNumSweeps = num_sweeps;
-          input.tolerance = norm_based > 1 ? tol : 0;
+          input.tolerance = norm_based ? tol : 0;
```

Comment:
Didn't we handle this in a separate PR?

Reply:
Yes. I include this for a testing purpose as the Andrew's PR is not yet merged. Once his PR is merged and I will include a merge commit.

Reply:
FYI, that PR looks like it's making the BlockTriDi test fail.